libs/dataset-explorer/src/lib/DatasetExplorerTab.tsx

```diff
@@ -304,68 +267,20 @@ export class DatasetExplorerTab extends React.Component<
     }
   };
 
-  private onChartTypeChange = (
-    _ev?: React.SyntheticEvent<HTMLElement>,
-    item?: IChoiceGroupOption
-  ): void => {
-    const newProps = _.cloneDeep(this.state.chartProps);
-    if (item?.key === undefined || !newProps) {
-      return;
-    }
-    newProps.chartType = item.key as ChartTypes;
-    if (newProps.yAxis.property === ColumnCategories.None) {
-      newProps.yAxis = this.generateDefaultYAxis();
-    }
-    this.setState({ chartProps: newProps });
-    this.logButtonClick(TelemetryEventName.DatasetExplorerNewChartTypeSelected);
-  };
-
   private logButtonClick = (eventName: TelemetryEventName): void => {
     this.props.telemetryHook?.({
```

Comment:
Do we want to update all props.telemetryHook to context.telemetryHook? For example this line?

Reply:
I am planning to wrap all buttons/input etc for automatically telemetry.